templates/azml-app/app/startupHelpers.py

```diff
@@ -0,0 +1,90 @@
+import requests
+import os
+from xml.etree import ElementTree as et
+
+from app.generateScore import ScoreFileGenerator
+
+
+def get_model_details():
+    """
+    This function retrieves the model from Azure ML and returns the model and tokenizer.
+    It uses the azcopy command to copy the model files from Azure Blob Storage to the local directory.
+    """
+    import xml.etree.ElementTree as et
+    import requests as rq
+    # Get the model id and path from environment variables
+    model_id = os.environ["AZURE_ML_MODEL_ID"]
+    model_path = os.environ["AZURE_ML_MODEL_PATH"]
+    if model_path is None or model_id is None:
+        raise ValueError("AZURE_ML_MODEL_PATH or AZURE_ML_MODEL_ID environment variable not set")
+    res = rq.post("https://ml.azure.com/api/eastus/assetstore/v1.0/dataReference/getBlobReferenceSAS", json={"assetId": model_id, "blobUri": model_path})

```

Comment:
>eastus [](http://example.com/codeflow?start=44&length=6)

do we always go to eastus, or should we make this dynamic? if this is difficult to do right now, we can open an issue to fix this later.

Reply:
will sync on this offline